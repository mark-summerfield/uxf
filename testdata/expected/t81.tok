FileComment Str("last modified 1 April 2001 by John Doe")
TClassBegin ttype=<Owner>
  Field Str("name") vtype=str
  Field Str("organization") vtype=str
TClassEnd
TClassBegin ttype=<Database> # use IP address in case network name resolution is not working
  Field Str("server") vtype=str
  Field Str("port") vtype=int
  Field Str("file") vtype=str
TClassEnd
ListBegin
  TableBegin ttype=<Owner>
    Str("John Doe")
    Str("Acme Widgets Inc.")
  TableEnd
  TableBegin ttype=<Database>
    Str("192.0.2.62")
    Int(143)
    Str("payroll.dat")
  TableEnd
ListEnd
Eof
----------------------------------------
