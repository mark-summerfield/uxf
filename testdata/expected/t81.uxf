uxf 1.0
#<last modified 1 April 2001 by John Doe>
=#<use IP address in case network name resolution is not working> Database server:str port:int
   file:str
=Owner name:str organization:str
[
  (Owner <John Doe> <Acme Widgets Inc.>)
  (Database <192.0.2.62> 143 <payroll.dat>)
]
