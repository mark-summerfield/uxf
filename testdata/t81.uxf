uxf 1.0
#<last modified 1 April 2001 by John Doe>
=Owner name:str organization:str
=#<use IP address in case network name resolution is not working>
  Database server:str port:int file:str
[
  (Owner <John Doe> <Acme Widgets Inc.>)
  (Database <192.0.2.62> 143 <payroll.dat>)
]
