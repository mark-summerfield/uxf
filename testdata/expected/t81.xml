<?xml version="1.0" ?>
<uxf version="1.0" comment="last modified 1 April 2001 by John Doe">
  <ttypes>
    <ttype name="Database" comment="use IP address in case network name resolution is not working">
      <field name="server" vtype="str"/>
      <field name="port" vtype="int"/>
      <field name="file" vtype="str"/>
    </ttype>
    <ttype name="Owner">
      <field name="name" vtype="str"/>
      <field name="organization" vtype="str"/>
    </ttype>
  </ttypes>
  <list>
    <table name="Owner">
      <row>
        <str>John Doe</str>
        <str>Acme Widgets Inc.</str>
      </row>
    </table>
    <table name="Database">
      <row>
        <str>192.0.2.62</str>
        <int v="143"/>
        <str>payroll.dat</str>
      </row>
    </table>
  </list>
</uxf>
